### PR TITLE
[Collectives] Deprecate active based language

### PR DESCRIPTION
**Pull request by kwaters4**
This is a mirror of the PR on the main specification. Deprecates all blocks with active based collectives and creates new text with team based collectives.

### CODE REVIEW

File: content/shmem_team_split_strided.tex [hunk 1]
```diff
@@ -101,11 +101,8 @@
 }
 
 \apinotes{
-  The \FUNC{shmem\_team\_split\_strided} operation uses an arbitrary
-  \VAR{stride} argument, whereas the \VAR{logPE\_stride} argument to the
-  active set collective operations only permits strides that are a power of two.
-  Arbitrary strides allow a greater number of PE subsets to be expressed
-  and can support a broader range of usage models.
+  The \FUNC{shmem\_team\_split\_strided} operation can take any positive integer value
```
Comment (davidozog) [inline content/shmem_team_split_strided.tex]:
```suggestion
  The \FUNC{shmem\_team\_split\_strided} operation can take any integer value
```

**Review by davidozog — COMMENTED** [review]
I think we can take this on as a section edit for v1.6.  After combing through, I only found 1 accidental semantic change regarding the team split stride value, everything else is just moving things around and appropriately wrapping text with deprecation markers.

It does bloat the length of the collectives sections a tiny bit, but also puts us in position to completely remove the deprecated text without any undesirable consequences (yay!).  Maybe we remove in v1.7?

**Review by davidozog — APPROVED** [review]

**Comment by davidozog** [comment]
This closes #452 - looking for 1 more review w/ approval @maawad @BKP @isubsmith

File: content/shmem_broadcast.tex [hunk 2]
```diff
@@ -75,66 +74,89 @@
...
+   	 \item The \dest{} array on all \acp{PE} participating in the broadcast
+         is ready to accept the broadcast data.
+   	 \item The \VAR{pSync} array on all \acp{PE} in the
+         active set is not still in use from a prior call to an \openshmem
+         collective routine.
     \end{itemize}
     Otherwise, the behavior is undefined.
 
-    Upon return from a broadcast routine, the following are true for the local
+	Upon return from a active-based broadcast routine, the following are true for the local
```
Comment (maawad) [inline content/shmem_broadcast.tex]:
```suggestion
	Upon return from an active-based broadcast routine, the following are true for the local
```

File: content/shmem_reductions.tex [hunk 3]
```diff
@@ -257,6 +257,8 @@ \subsubsubsection{PROD}
     \VAR{nreduce} must be of type integer.}
 
 \begin{DeprecateBlock}
+\apiargument{IN}{nreduce}{In active-set based \ac{API} calls,
```
Comment (maawad) [inline content/shmem_reductions.tex:258]: I can't comment on the above line but should we change the above sentence from:
```
\apiargument{IN}{nreduce}{The number of elements in the \dest{} and \source{}
    arrays. In teams based \ac{API} calls, \VAR{nreduce} must be of type size\_t.
    In deprecated active-set based \ac{API} calls,
    \VAR{nreduce} must be of type integer.}
```
to
```
\apiargument{IN}{nreduce}{The number of elements in the \dest{} and \source{}
    arrays. In teams based \ac{API} calls, \VAR{nreduce} must be of type size\_t.}
```

**Review by maawad — APPROVED** [review]
Left a couple of comments but otherwise looks good.

File: content/shmem_reductions.tex (hunk 3 above)
Comment (kwaters4) [reply content/shmem_reductions.tex:258]: Resolved, Nice catch

**Review by davidozog — APPROVED** [review]

**Pull request merged**